src/pages/ct-default/form/CtDefaults.js

```diff
@@ -70,54 +62,38 @@ const CtDefaults = ({ _labels, access }) => {
   })
 
   const getDataResult = () => {
-    var parameters = `_filter=`
-    getRequest({
-      extension: CurrencyTradingSettingsRepository.Defaults.qry,
-      parameters: parameters
+    const myObject = {}
+    defaultsData.list.forEach(obj => {
+      if (arrayAllow.includes(obj.key)) {
+        myObject[obj.key] = obj.key ? parseInt(obj.value) : null
+        formik.setFieldValue(obj.key, parseInt(obj.value))
+      }
     })
-      .then(res => {
-        const myObject = {}
-        res.list.forEach(obj => {
-          if (arrayAllow.includes(obj.key)) {
-            myObject[obj.key] = obj.key ? parseInt(obj.value) : null
-            formik.setFieldValue(obj.key, parseInt(obj.value))
-          }
-        })
-
-        if (myObject && myObject['ct-nra-individual']) {
-          getNumberRange(myObject['ct-nra-individual'], 'ct-nra-individual')
-        }
-
-        if (myObject && myObject['ct-nra-corporate']) {
-          getNumberRange(myObject['ct-nra-corporate'], 'ct-nra-corporate')
-        }
-      })
-      .catch(error => {
-        setErrorMessage(error)
-      })
+    if (myObject && myObject['ct-nra-individual']) {
+      getNumberRange(myObject['ct-nra-individual'], 'ct-nra-individual')
+    }
+    if (myObject && myObject['ct-nra-corporate']) {
+      getNumberRange(myObject['ct-nra-corporate'], 'ct-nra-corporate')
+    }
   }
 
   const getNumberRange = (nraId, key) => {
     var parameters = `_filter=` + '&_recordId=' + nraId
     getRequest({
       extension: SystemRepository.NumberRange.get,
       parameters: parameters
```

Comment:
please assign `_filter=` + '&_recordId=' + nraId' directly to request parameters no need for var parameter